### PR TITLE
Update BoilerplateProjects.md

### PR DESCRIPTION
**Pull request by amiya-pattnaik**
Updated the Boilerplate repos to V8 and added links to the specific repos for community use,

## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/docs/BoilerplateProjects.md [hunk 1]
```diff
@@ -29,6 +29,46 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
+- Features:
+    - Page Objects Model uses with ES6 /ES7 style class base approach and TypeScript support
+    - Examples of multi selector option to query element with more than one selector at a time
+    - Examples of multi browser and headless browser execution using - Chrome and Firefox
+    - Cloud testing Integration with BrowserStack, Sauce Labs, LambdaTest
+    - Examples of read/write data from MS-Excel for easy test data management from external data sources with examples
+    - Database support to any RDBMS (Oracle, MySql, TeraData, Vertica etc.), executing any queries / fetching result set etc. with examples for E2E testing
+    - Multiple reporting (Spec, Xunit/Junit, Allure, JSON) and Hosting Allure and Xunit/Junit reporting on WebServer.
+    - Examples with demo app https://search.yahoo.com/  and http://the-internet.herokuapp.com.
+    - BrowserStack, Sauce Labs, LambdTest and Appium specific .config file ( for playback on mobile device.)` For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:44]:
```suggestion
    - BrowserStack, Sauce Labs, LambdTest and Appium specific `.config` file (for playback on mobile device). For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX).
```

File: website/docs/BoilerplateProjects.md [hunk 2]
```diff
@@ -58,6 +98,10 @@ This boilerplate includes the following:
         - Swipe
         - Browsers
 
+## [amiya-pattnaik/webdriverIO-with-cucumberBDD-V7](https://github.com/amiya-pattnaik/webdriverIO-with-cucumberBDD/tree/wdio-v7)
+## [amiya-pattnaik/webdriverIO-with-mochaBDD-V7](https://github.com/amiya-pattnaik/webdriverIO-with-mochaBDD/tree/wdio-v7)
+## [amiya-pattnaik/webdriverIO-with-jasmineBDD-V7](https://github.com/amiya-pattnaik/webdriverIO-with-jasmineBDD/tree/wdio-v7)
+
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:104]: No need to advertise v7 boilerplates.
```suggestion
```

File: website/docs/BoilerplateProjects.md [hunk 3]
```diff
@@ -29,6 +29,46 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
+- Framework: WDIO-V8 with Mocha (V10x).
+- Features:
+    -  Page Objects Model uses with ES6 /ES7 style class base approach and TypeScript support
+    -  Examples with demo app https://search.yahoo.com  and http://the-internet.herokuapp.com
+    -  Examples of multi browser and headless browser execution using - Chrome and Firefox
+    -  Cloud testing Integration with BrowserStack, Sauce Labs, LambdaTest
+    -  Multiple reporting (Spec, Xunit/Junit, Allure, JSON) and Hosting Allure and Xunit/Junit reporting on WebServer.
+    -  Examples of read/write data from MS-Excel for easy test data management from external data sources with examples
+    -  Examples of DB connect to any RDBMS (Oracle, MySql, TeraData, Vertica etc.), any query execution / fetching result set etc. with examples for E2E testing
+    -  BrowserStack, Sauce Labs, LambdTest and Appium specific .config file ( for playback on mobile device.)` For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:57]:
```suggestion
    -  BrowserStack, Sauce Labs, LambdTest and Appium specific `.config` file (for playback on mobile device). For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX).
```

File: website/docs/BoilerplateProjects.md [hunk 4]
```diff
@@ -29,6 +29,46 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
+- Framework: WDIO-V8 with Jasmine (V4x).
+- Features:
+    -  Page Objects Model uses with ES6 /ES7 style class base approach and TypeScript support
+    -  Examples with demo app https://search.yahoo.com  and http://the-internet.herokuapp.com
+    -  Examples of multi browser and headless browser execution using - Chrome and Firefox
+    -  Cloud testing Integration with BrowserStack, Sauce Labs, LambdaTest
+    -  Multiple reporting (Spec, Xunit/Junit, Allure, JSON) and Hosting Allure and Xunit/Junit reporting on WebServer.
+    -  Examples of read/write data from MS-Excel for easy test data management from external data sources with examples
+    -  Examples of DB connect to any RDBMS (Oracle, MySql, TeraData, Vertica etc.), any query execution / fetching result set etc. with examples for E2E testing
+     -  BrowserStack, Sauce Labs, LambdTest and Appium specific .config file ( for playback on mobile device.)` For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:70]:
```suggestion
     -  BrowserStack, Sauce Labs, LambdTest and Appium specific `.config` file ( for playback on mobile device). For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX).
```

**Comment by christian-bromann** [comment]
@amiya-pattnaik once the CLA is signed I can go ahead and merge the PR.

**Comment by amiya-pattnaik** [comment]
Hi Chris,
CLA  signed is completed. Please check and do the needful.
Thanks,

**Comment by christian-bromann** [comment]
@amiya-pattnaik why did you close the PR?

**Comment by amiya-pattnaik** [comment]
@christian-bromann 
I reopen the PR. not sure how it happened. I guess accidentally I hit the `Close with comment` instead of comment.
Thanks,
AP

File: website/docs/BoilerplateProjects.md [hunk 5]
```diff
@@ -29,6 +29,46 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
+- Framework: WDIO-V8 with Jasmine (V4x).
+- Features:
+    -  Page Objects Model uses with ES6 /ES7 style class base approach and TypeScript support
+    -  Examples with demo app https://search.yahoo.com  and http://the-internet.herokuapp.com
+    -  Examples of multi browser and headless browser execution using - Chrome and Firefox
+    -  Cloud testing Integration with BrowserStack, Sauce Labs, LambdaTest
+    -  Multiple reporting (Spec, Xunit/Junit, Allure, JSON) and Hosting Allure and Xunit/Junit reporting on WebServer.
+    -  Examples of read/write data from MS-Excel for easy test data management from external data sources with examples
+    -  Examples of DB connect to any RDBMS (Oracle, MySql, TeraData, Vertica etc.), any query execution / fetching result set etc. with examples for E2E testing
+     -  BrowserStack, Sauce Labs, LambdTest and Appium specific `.config` file ( for playback on mobile device). For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX).
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:70]:
```suggestion
    -  BrowserStack, Sauce Labs, LambdTest and Appium specific `.config` file ( for playback on mobile device). For one click Appium setup on local machine for iOS and Android refer to [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX).
```

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**